src/views/Home.vue

```diff
@@ -6,7 +6,10 @@
 
 <script>
 export default {
-  name: 'Home',
+  name : "Home",
+  mounted() {
+    console.log( process.env.VUE_APP_fuck );
```

Comment:
A little NSFW, don't you think?

Reply:
LoL, was just testing env variables. Just making a followup commit 😅